### PR TITLE
Added Project registry and namespace info to doc

### LINKED ISSUE #1564: Document card view for images in repositories
Per https://github.com/vmware/harbor/issues/4249

### LINKED ISSUE #1597: Document registry namespaces
I can't find details about this in vic-planning, but @alextopuzov obtained confirmation from @lazarin that this is a feature in 1.4.

### PR DESCRIPTION
**Pull request by alextopuzov**
Fixes #1597 

@gmuleshkov or @shadjiiski can you verify whether the new additions are technically correct and cover what the users should know about Project registries and project namespaces?

Thanks.

### CODE REVIEW

File: docs/user_doc/vic_cloud_admin/add_repos_in_portal.md [hunk 1]
```diff
@@ -17,16 +19,19 @@ Protocol | Description
...
+2. In the add registry dialog box, configure the registry settings.
+	1. As address, enter the IP or hostname of the registry, the port, and optionally a namespace.
+
+	For example: `https://registry.hub.docker.com:443/vmware`
 
-5. In the add registry dialog box, configure the registry settings.
-	1. As address, enter the IP or hostname of the registry and the port.
 	2. Enter name for the registry.
-	3. Select the login credential and click **Verify**.
+	3. Optionally, select the login credentials to access the registry and click **Verify**.
```
Comment (shadjiiski) [inline docs/user_doc/vic_cloud_admin/add_repos_in_portal.md]: > Optionally, select the login credentials to access the registry and click **Verify**

Refer to the comment for `add_project_registry.md` - same comment here

**Comment by shadjiiski** [comment]
@alextopuzov, thanks for the pull request! I have added some comments (see review above this comment). Please do not hit the Merge button if you get all green. Unless something has changed recently, we will need to get your change through the internal pipeline in order to merge it successfully. cc @lazarin

File: docs/user_doc/vic_cloud_admin/add_project_registry.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+1. In the management portal, navigate to **Administration** > **Projects** and click your project.
+
+2. On the Project Registries tab, click **+New Project Registry**.
+3. On the Add Project Registry page, configure your new registry.
+	1. As address, enter the IP or hostname of the registry, the port, and optionally a namespace.
+
+	For example: `https://registry.hub.docker.com:443/library`
+
+	2. Enter name for the registry.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/add_project_registry.md]: Enter a name for the registry.

File: docs/user_doc/vic_cloud_admin/add_project_registry.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	3. Optionally, select the login credentials to access the registry and click **Verify**.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/add_project_registry.md]: Do we need to add some info about what these credentials are?

File: docs/user_doc/vic_cloud_admin/add_repos_in_portal.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
+You can add multiple registries, in addition to the integrated vSphere Integrated Containers Registry to gain access to both public and private images. You can enable and disable the registries that you added. You can add global registries, visible for all projects and project registries available only for the project for which they are added. When you disable a registry, searching for templates and images in that registry is disabled. Even if you disable the default https://registry.hub.docker.com registry, you can still access the popular templates. To customize your popular templates, see [Customize the Popular Templates list](https://github.com/vmware/admiral/wiki/Configuration-guide#customize-the-popular-templates-list) documentation.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/add_repos_in_portal.md]: "You can add global registries, visible for all projects and project registries available only for the project for which they are added." ==>

"You can add global registries, that are visible to all projects, and project registries, that are available only to the project to which they are added.

File: docs/user_doc/vic_cloud_admin/add_repos_in_portal.md [hunk 5]
```diff
@@ -17,16 +19,19 @@ Protocol | Description
 
 **Procedure**
 
-1. In the management portal, navigate to **Administration** > **Registries** > **Source Registries** and click **+Registry**.
+1. In the management portal, navigate to **Administration** > **Global Registries** > **Source Registries** and click **+Registry**.
+
+2. In the add registry dialog box, configure the registry settings.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/add_repos_in_portal.md]: "add registry" with upper case letters?

File: docs/user_doc/vic_cloud_admin/add_repos_in_portal.md [hunk 6]
```diff
@@ -17,16 +19,19 @@ Protocol | Description
...
+	1. As address, enter the IP or hostname of the registry, the port, and optionally a namespace.
+
+	For example: `https://registry.hub.docker.com:443/vmware`
 
-5. In the add registry dialog box, configure the registry settings.
-	1. As address, enter the IP or hostname of the registry and the port.
 	2. Enter name for the registry.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/add_repos_in_portal.md]: Enter a name

File: docs/user_doc/vic_overview/roles_and_personas.md [hunk 7]
```diff
@@ -30,6 +30,7 @@ Role assigned per project. In addition to the view access, for their assigned pr
 Role assigned per project. For their assigned projects, DevOps administrators can perform additional actions in vSphere Integrated Containers Management Portal:
 
 - Add developers and viewers and assign other DevOps administrators
+- Add new registries for their project
```
Comment (stuclem) [inline docs/user_doc/vic_overview/roles_and_personas.md]: Add new registries to their project

File: docs/user_doc/vic_cloud_admin/add_project_registry.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# Add Project Specific Registries #
+
+In addition to the integrated vSphere Integrated Containers Registry and the global registries added by the Cloud administrator, DevOps administrators can add project specific registries.
+
+Starting with vSphere Integrated Containers 1.4, you can also configure namespaces for the registries that you add. If you add a new registry and configure a namespace for it, developers cannot search, browse, or deploy images that are outside of that namespace. You can add a registry multiple times to allow developers to reach different namespaces in that registry.
```
Comment (gmuleshkov) [inline docs/user_doc/vic_cloud_admin/add_project_registry.md]: Not really sure about it but may be we can mention that the registries for a project are the system registries + the project registries. If a registry is added as global it is accessible to all projects

File: docs/user_doc/vic_cloud_admin/add_project_registry.md (hunk 3 above)
Comment (alextopuzov) [reply docs/user_doc/vic_cloud_admin/add_project_registry.md]: I wouldn't go that far.

**Comment by alextopuzov** [comment]
@shadjiiski all your comments, apart from whitelist are addressed. I have another issue that will address whitelist.

@gmuleshkov deleting registries is too straightforward to document. I've only added the information about DevOps admins being able to see global registries with no way of deleting them.

File: docs/user_doc/vic_cloud_admin/add_project_registry.md (hunk 3 above)
Comment (stuclem) [reply docs/user_doc/vic_cloud_admin/add_project_registry.md]: Fair enough

**Pull request merged**